src/actions/tags.js

```diff
@@ -112,10 +112,16 @@ export const clearSearchString = () => {
 * Action to reset all tags to empty
 */
 
-export const resetTags = (displayName, id) => {
```

Comment:
ah yes we need to remove that.  But the action doesn't use any of the arguments so it doesn't matter (https://github.com/numo-labs/isearch-ui/blob/master/src/actions/tags.js#L114) but we should definitely remove it from the file!!